### ISSUE TITLE
Text goes over image in profile page

### ISSUE THREAD

**Issue by talovdat** [Way2CU/IBI-Web-Site#56]
The 'APPROACH' article goes over the image on the profile page.

![image](https://user-images.githubusercontent.com/26923359/34648530-84c4d6a0-f3a3-11e7-8457-1c7ef50775f9.png)

**Comment by MeanEYE** [comment]
Perhaps on small screens. That's by design so that text remains visible at all times instead of being cut off. Check on regular screen.

**Comment by talovdat** [comment]
On laptop screen it looks normal.
This issue applies to bigger screens.

**Comment by MeanEYE** [comment]
Hm. I'll have it tested.

**Comment by MeanEYE** [comment]
@VladimirWebDev make sure this is the case. Test different sizes and let me know why it's happening. Target content size is 1200, so screens should never be smaller than that.

**Comment by VladimirWebDev** [comment]
First thing, I made mistake not testing bigger resolution when image is changed (since it's changed). On devices < 1200px there is no problem like this since text comes first than image is displayed (However I make sure now that everything is displayed correctly). Tested on Apple iPad Air 2 emulator on FF and on my tablet. On screen size 1920x1080 problem persist. Image is set as `background-image`. Problem I can see is size and position not set adequately.
